project_status.py

```diff
@@ -215,108 +211,449 @@ def parse_qualimap(genome_results_file, sample):
+                if sample in result:
+                    result[sample] = [result[sample]]
+                    result[sample].append('Empty exit code, but sample is not under analysis')
+                else:
+                    result[sample] = 'Empty exit code, but sample is not under analysis'
+            elif exit_code.strip() != '0' and exit_code != '':
+                if sample in result:
+                    result[sample] = [result[sample]]
+                    result[sample].append('Exit code: {}'.format(exit_code))
+                else:
+                    result[sample] = 'Exit code: {}'.format(exit_code)
+    return result
+
+def get_incoherent_samples(results_francesco):
+    result = {}
+    for sample_id, sample in results_francesco.items():
+        sequenced = sample.get('#Archived_runs', '')
+        organized = sample.get('#Data_runs', '')
+        analyzed = sample.get('#Analysis_runs', '')
+        try:
+            sequenced = int(sequenced)
+            organized = int(organized)
+            analyzed = int(analyzed)
+        # if not int or something strange in the results, print it too (just in case)
+        except ValueError, e:
+            result[sample_id] = {'sequenced': sequenced, 'organized': organized, 'analyzed': analyzed}
+        else:
+            if not(sequenced == organized == analyzed):
+                result[sample_id] = {'sequenced': sequenced, 'organized': organized, 'analyzed': analyzed}
+    return result
+
+
+def get_sequenced(project):
+    incoming = "/proj/ngi2016003/incoming"
+    project_flowcells = {}
+    for fc in os.listdir(incoming):
+        sample_sheet = os.path.join(incoming, fc, 'SampleSheet.csv')
+        command = 'grep {} {}'.format(project, sample_sheet)
+        try:
+            p = subprocess.Popen(command, shell=True, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except Exception, e:
+            print 'Command failed: {}'.format(command)
+            raise e
+        output = p.communicate()[0]
+        if output:
+            for line in output.split('\n'):
+                # if not an empty line
+                if line:
+                    try:
+                        sample = line.split(',')[2]
+                    except Exception, e:
+                        print line
+                        print 'Skipping line: {} from sample sheet: {}'.format(line, sample_sheet)
+                        # if something went wrong
+                        continue
+                    else:
+                        if sample not in project_flowcells:
+                            project_flowcells[sample] = [fc]
+                        else:
+                            project_flowcells[sample].append(fc)
+    return project_flowcells
+
+def get_organized(project):
+    sequenced = get_sequenced(project)
+    project_path = os.path.join('/proj/ngi2016003/nobackup/NGI/DATA', project)
```

Comment:
this can be done looking at the samples hash, maybe just adding some more info there